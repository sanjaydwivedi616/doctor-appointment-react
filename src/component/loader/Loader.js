import React from 'react'

const Loader = () => {
    return (
        <div>
            <p className="text-center">Loadding....</p>
        </div>
    )
}

export default Loader;