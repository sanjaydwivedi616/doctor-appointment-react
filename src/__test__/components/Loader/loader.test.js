import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../../../component/loader/Loader';

it('When Loader is render', () => {
  const loaderText = renderer.create(<Loader />)
  expect(loaderText).toMatchSnapshot();
});