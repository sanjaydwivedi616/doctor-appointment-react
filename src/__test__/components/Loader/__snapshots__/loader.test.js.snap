// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When Loader is render 1`] = `
<div>
  <p
    className="text-center"
  >
    Loadding....
  </p>
</div>
`;
